src/routes/MessageRoutes.ts

```diff
@@ -0,0 +1,20 @@
+import { Router } from 'express';
+import { inject, injectable } from 'tsyringe';
+import { getUser, isVerified } from './middleware';
+import { MessageController } from '../controllers/MessageController';
+
+@injectable()
+export class MessageRoutes {
+	private readonly messageController: MessageController;
+
+	public constructor(@inject(MessageController) messageController: MessageController) {
+		this.messageController = messageController;
+	}
+
+	public routes(router: Router): void {
+		router.post('/channels/:channelID/messages', getUser, isVerified, this.messageController.createMessage.bind(this.messageController));
+		router.get('/channels/:channelID/messages', getUser, isVerified, this.messageController.getMessages.bind(this.messageController));
+		router.get('/channels/:channelID/messages/:messageID', getUser, isVerified, this.messageController.getMessage.bind(this.messageController));
+		router.delete('/channels/:channelID/messages/:messageID', getUser, isVerified, this.messageController.deleteMessage.bind(this.messageController));
```

Comment:
Question: Do you have the concept of "restricted channels", i.e. channels that can only be accessed by admins?

Reply:
As of now, not planning on any restricted channels. The other ones I'm thinking of for now are DM channels and potentially the temporary 1-to-1 channels (although they might be implemented under DM channels too)